src/genegraph/sink/base.clj

```diff
@@ -46,10 +47,16 @@
     (log/info :fn :import-documents! :msg :importing :name (:name d))
     (db/load-model (transform-doc d) (:name d))))
 
+(defn build-suggesters! []
+  (log/info :fn :build-suggesters! :msg :building)
+  (build-all-suggestions)
+  (log/info :fn :build-suggesters! :msg :complete))
+
 (defn initialize-db! []
   (let [res (read-base-resources)]
     (retrieve-base-data! res)
     (import-documents! res)
+    (build-suggesters!)
     (log/info :fn :initialize-db! :msg :initialization-complete)))
```

Comment:
Do the suggesters depend on having the curations available? This wouldn't be true at this stage...

Reply:
That explains why I'm not seeing curations when I create-migration. Thanks for pointing this out.